frontend/user/templates/user.html

```diff
@@ -3,33 +3,51 @@
 {%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
 
 {% block content %}
-  <div>
-      <h1 class="govuk-heading-xl">Apply for Funding</h1>
-  </div>
 <div>
+    <h1 class="govuk-heading-xl">Apply for Funding</h1>
+</div>
+<div>
+
+    {% if roles_error %}
+    <p class="govuk-body">You do not have access
+    <p class="govuk-body">Either you:</p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li class="govuk-body">
+            do not have the right account permissions set up, or
+        </li>
+        <li class="govuk-body">
+            the link you’re using has expired
```

Comment:
Did we ever discover why this text is here? As no-one will be here from an expired link?

Reply:
edited now after confirming with peter 👍🏾